Pkgfile

```diff
@@ -1,9 +1,9 @@
-# syntax = ghcr.io/siderolabs/bldr:v0.3.1
+# syntax = ghcr.io/siderolabs/bldr:v0.3.2
 
 format: v1alpha2
 
 vars:
-  LINUX_FIRMWARE_VERSION: "20240513" # update this when updating PKGS_VERSION in Makefile
+  LINUX_FIRMWARE_VERSION: "20240709" # update this when updating PKGS_VERSION in Makefile
   DRBD_DRIVER_VERSION: 9.2.9 # update this when updating PKGS_VERSION in Makefile
   ZFS_DRIVER_VERSION: 2.2.4 # update this when updating PKGS_VERSION in Makefile
   UTIL_LINUX_VERSION: 2.40.1 # update this when updating PKGS_VERSION in Makefile
```

Comment:
I think this got updated in `pkgs` as well?

Reply:
Yep, thanks!